docs/turbo/generators/utils.ts

```diff
@@ -0,0 +1,64 @@
+import fetch from "node-fetch";
+
+interface Answers extends Object {
+  turboStars: string;
+  turboDownloads: string;
+  turboYearsSaved: string;
+}
+
+const PUBLIC_TB_TOKEN =
+  "p.eyJ1IjogIjAzYzA0Y2MyLTM1YTAtNDhhNC05ZTZjLThhMWE0NGNhNjhkZiIsICJpZCI6ICJmOWIzMTU5Yi0wOTVjLTQyM2UtOWIwNS04ZDZlNzIyNjEwNzIifQ.A3TOPdm3Lhmn-1x5m6jNvulCQbbgUeQfAIO3IaaAt5k";
+
+export async function releasePostStats(answers: Answers): Promise<string> {
+  const [starsResponse, downloadsResponse, timeSavedResponse] =
+    await Promise.all([
+      fetch("https://api.github.com/repos/vercel/turbo"),
+      fetch("https://api.npmjs.org/versions/turbo/last-week"),
+      fetch(
+        `https://api.us-east.tinybird.co/v0/pipes/turborepo_time_saved_ticker.json?token=${PUBLIC_TB_TOKEN}`
+      ),
+    ]);
+
+  const [starsData, downloadsData, timeSavedData] = await Promise.all([
+    starsResponse.json() as { stargazers_count: number },
+    downloadsResponse.json() as {
+      downloads: Array<{ [key: string]: number }>;
+    },
+    timeSavedResponse.json() as {
+      data: [
+        {
+          remote_cache_minutes_saved: number;
+          local_cache_minutes_saved: number;
+        }
+      ];
+    },
+  ]);
+
+  const totalMinutesSaved: number =
+    timeSavedData.data[0].remote_cache_minutes_saved +
+    timeSavedData.data[0].local_cache_minutes_saved;
+  const totalYearsSaved: number = Math.floor(totalMinutesSaved / 60 / 24 / 365);
```

Comment:
I find grouping the divisors a little more readable. Maybe we should just have a `MINUTES_IN_YEAR` constant
```suggestion
  const totalYearsSaved: number = Math.floor(totalMinutesSaved / (60 * 24 * 365));
```

Reply:
👍🏼 good call, will add in a follow up so I don't have to suffer through the rust rebuild from the lockfile change 😭